common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java

```diff
@@ -55,8 +55,10 @@
  */
 @SuppressWarnings("PMD.AbstractClassShouldStartWithAbstractNamingRule")
 public abstract class GrpcClient extends RpcClient {
-    
+
```

Comment:
Please do not change the indent. You can use nacosCodeStyle in source code to reformat codes.

Reply:
Got it,I pushed a formatted version.

Reply:
Got it,I submitted a formatted version